scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2738,10 +2674,12 @@ SCX_OPS_DEFINE(layered,
 	       .yield			= (void *)layered_yield,
 	       .set_weight		= (void *)layered_set_weight,
 	       .set_cpumask		= (void *)layered_set_cpumask,
+	       .update_idle		= (void *)layered_update_idle,
 	       .cpu_release		= (void *)layered_cpu_release,
 	       .init_task		= (void *)layered_init_task,
 	       .exit_task		= (void *)layered_exit_task,
 	       .dump			= (void *)layered_dump,
 	       .init			= (void *)layered_init,
 	       .exit			= (void *)layered_exit,
+	       .flags			= SCX_OPS_KEEP_BUILTIN_IDLE,
```

Comment:
Is this flag available for backports as well?

Reply:
Yeah, that flag has been around for a long time.